vmware/nsx/server_templates/NSX - Firewall Demo/RL10_Linux_Enable_Managed_Login.sh

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
```

Comment:
can the rightlink scripts point to the real ones?

Reply:
Done